src/Config/Entity.cs

```diff
@@ -594,17 +600,17 @@ public record RestStoredProcedureEntitySettings([property: JsonPropertyName("met
     /// instead of using the entity-name. Can be a string type.
     /// </param>
     /// <param name="RestMethods">Defines the HTTP actions that are supported for stored procedures.</param>
-    public record RestStoredProcedureEntityVerboseSettings(object? Path,
-                                     [property: JsonPropertyName("methods")] RestMethod[]? RestMethods = null);
+    public record RestStoredProcedureEntityVerboseSettings([property: JsonPropertyName(RestEntitySettings.PROPERTY_PATH)] object? Path,
+                                     [property: JsonPropertyName(RestStoredProcedureEntitySettings.PROPERTY_METHODS)] RestMethod[]? RestMethods = null);
 
     /// <summary>
     /// Describes the GraphQL settings specific to an entity.
     /// </summary>
     /// <param name="Type">Defines the name of the GraphQL type.
     /// Can be a string or Singular-Plural type.
     /// If string, a default plural route will be added as per the rules at
-    /// </param>
     /// <seealso cref="<https://engdic.org/singular-and-plural-noun-rules-definitions-examples/"/>
+    /// </param>
```

Comment:
Does IntelliSense still detect `<seealso />` be nested inside `<param></param>`?

Reply:
yes